### PR TITLE
chore(common): CHECKOUT-6591 Set node version to 14 for repo

### PR DESCRIPTION
**Pull request by animesh1987**
## What?
Update node version of sdk

## Why?
Before we start the next project, it's better to update to a newer node version

## Testing / Proof
- Tests

@bigcommerce/checkout

### CODE REVIEW

**Comment by TomA-R** [comment]
Could you update .nvmrc too?

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-10
+14
```
Comment (davidchin) [inline .nvmrc:1]:
```suggestion
14.18
```

To match with the version we're using in CircleCI

File: package.json [hunk 2]
```diff
@@ -18,9 +18,13 @@
...
     "url": "https://github.com/bigcommerce/checkout-sdk-js/issues"
   },
+  "engines": {
+    "node": "14.18",
+    "npm": "6"
+  },
   "homepage": "https://github.com/bigcommerce/checkout-sdk-js",
   "scripts": {
-    "prepare": "check-node-version --node '>=6' --npm '>=6'",
+    "prepare": "check-node-version --node '>=14' --npm '>=6'",
```
Comment (davidchin) [inline package.json:27]:
```suggestion
    "prepare": "check-node-version --node '>=14.18' --npm '>=6'",
```

File: src/payment/strategies/moneris/moneris-payment-strategy.spec.ts [hunk 3]
```diff
@@ -246,6 +246,24 @@ describe('MonerisPaymentStrategy', () => {
     });
 
     describe('#execute', () => {
+        it('throws error when moneris response is not successful', async () => {
```
Comment (davidchin) [inline src/payment/strategies/moneris/moneris-payment-strategy.spec.ts:249]: Do we need this change? It seems you just moved the `it` block to the top?

**Review by davidchin — APPROVED** [review]

File: src/payment/strategies/moneris/moneris-payment-strategy.spec.ts (hunk 3 above)
Comment (animesh1987) [reply src/payment/strategies/moneris/moneris-payment-strategy.spec.ts:249]: Yes there is an issue how these tests are written, as postmessage payload was not leaking through tests and it was having intermittent behaviour. So I moved this one to top, and that fixed the issue. Once this is merged I will ping the integrations team to have a look as to what exactly was causing the issue with the test.

**Pull request merged**